src/transformers/models/canine/configuration_canine.py

```diff
@@ -61,6 +61,9 @@ class CanineConfig(PretrainedConfig):
             The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
         layer_norm_eps (`float`, *optional*, defaults to 1e-12):
             The epsilon used by the layer normalization layers.
+        pad_token_id (`int`, *optional*, defaults to 0): Padding token id.
+        bos_token_id (`int`, *optional*, defaults to 57344): Beginning of stream token id.
+        eos_token_id (`int`, *optional*, defaults to 57345): End of stream token id.
```

Comment:
Let's make the description, like `Padding token id.` etc., in a separate line with necessary indent.

Reply:
Thank you for reviewing, @ydshieh. I have made the changes.